src/main/java/uk/gov/justice/hmpps/casenotes/services/CaseNoteService.java

```diff
@@ -68,11 +68,12 @@ public Page<CaseNote> getCaseNotes(final String offenderIdentifier, final CaseNo
 
         final List<CaseNote> sensitiveCaseNotes;
 
+        final var includeSensitiveCaseNotes = isAllowedToViewOrCreateSensitiveCaseNote() && caseNoteFilter.getIncludeSensitive();
```

Comment:
So now if a client is allowed to see sensitive case notes, they can be explicitly excluded by passing `includeSensitive: false` in the filter.  By default `includeSensitive` is true so remains backward compatible and unnoticeable to existing clients.